enhancements/scorecard-storage.md

```diff
@@ -0,0 +1,250 @@
+to perform data management of this test output directory as they require.
+
+### Test Plan
+
+**Note:** *Section not required until targeted at a release.*
+
+Consider the following in developing a test plan for this enhancement:
+- Will there be e2e and integration tests, in addition to unit tests?
+- How will it be tested in isolation vs with other components?
+
+No need to outline all of the test cases, just the general strategy. Anything
+that would count as tricky in the implementation and anything particularly
+challenging to test should be called out.
+
+All code is expected to have adequate tests (eventually with coverage
+expectations).
+
+### Graduation Criteria
+
+**Note:** *Section not required until targeted at a release.*
+
+Define graduation milestones.
+
+These may be defined in terms of API maturity, or as something else. Initial proposal
+should keep this high-level with a focus on what signals will be looked at to
+determine graduation.
+
+Consider the following in developing the graduation criteria for this
+enhancement:
+- Maturity levels - `Dev Preview`, `Tech Preview`, `GA`
+- Deprecation
+
+Clearly define what graduation means.
+
+#### Examples
+
+These are generalized examples to consider, in addition to the aforementioned
+[maturity levels][maturity-levels].
+
+##### Dev Preview -> Tech Preview
+
+- Ability to utilize the enhancement end to end
+- End user documentation, relative API stability
+- Sufficient test coverage
+- Gather feedback from users rather than just developers
+
+##### Tech Preview -> GA 
+
+- More testing (upgrade, downgrade, scale)
+- Sufficient time for feedback
+- Available by default
+
+**For non-optional features moving to GA, the graduation criteria must include
+end to end tests.**
+
+
+
+### Upgrade / Downgrade Strategy
+
+This feature doesn't impact existing scorecard tests or test configurations.
+
+I currently don't see value in backporting this feature to previous versions of the operator-sdk.
+
+### Version Skew Strategy
+
+
+## Implementation History
+
+
+
+## Drawbacks
+
+Using emptyDir volumes means the on-cluster volumes are not accessible when the test pod is removed.  However, there is no current requirement to have 
+volumes live beyond the life of the test Pod.
+
+## Alternatives
+
+### Alternative 1
+An alternative to this feature would be to design a means of embedding test content into the test Pod log itself, co-mingled with the current scorecard test result JSON.
+
+This seems really complicated and hard to maintain so I rejected this in favor of this proposal.
+
+### Alternative 2
+
+Another alternative to using emptyDir volumes would be to provision
+persistent volumes per test.  The downside of this approach is
+that you have to have persistent storage available on the cluster
+and you might have to manually manage error cases that would leave storage
+volumes on-cluster.
```

Comment:
probably worth noting that the api proposed in this EP could be extended in the future to support PVs if it becomes necessary.

Reply:
added that note just now.